docs/MO_DG/prepare_model/FP16_Compression.md

```diff
@@ -10,11 +10,11 @@ To compress the model, use the `--data_type` option:
 ```
 
 > **NOTE**: Using `--data_type FP32` will give no result and will not force `FP32` 
```

Comment:
I cannot change the heading here. I think the heading should be "Compressing a Model to FP16".

Reply:
Done.